### DISCUSSION TITLE
Introductions 🤝

### DISCUSSION THREAD

**Discussion by javalosr2004** [hack4impact-calpoly/starter-pack#15] [General]
Welcome to the Starter Pack Discussions!

To kick things off, please introduce yourself in this thread. Here's a simple template to guide you, but feel free to share as little or as much as you're comfortable with:

- **Name:** [Your Name] (Pronouns)
- **Hometown:** [City,  Country]
- **Year & Major:** [e.g "I'm a  3rd year transfer in Computer Science" or "I'm a freshman in Graphic Communications"]
- **Fun Fact:** [Something interesting about you, e.g., "I play the ukulele!" or "I can solve a Rubik's cube in under a minute!"]
- **Interests:** [Your hobbies, goals, and aspirations eg., "I love hiking and the outdoors." or "I'm Radiant in Valorant!" ]

Board Introductions
-------------------------------------------------------------

Hi, my name is Jesus Avalos. I'm from Santa Cruz, and I'll be one of your Executive Directors for the following school year. I am currently a 3rd year Computer Science major and I have a couple of hobbies ranging from going to the gym, hiking, reading books and coding for fun side projects. 

Hello, my name is Muzart Tuman, I am from San Ramon and I will be your Director of Engineering for this 2024-2025 school year. I am a 3rd year Computer Science major and for hobbies I enjoy playing soccer, going to the gym, and coding random fun projects.

Hi! I'm Sean Nguyen (he/him/his), and I'm one of the Associate Directors of Engineering this year! I'm originally from Orange County, and this is my 3rd year as a Computer Science major. I love swimming, scuba diving, playing the piano, and traveling the world!

Hi, my name is Taran Singh. I am from Huntington Beach. I am currently a Sophomore Computer Science Major, and I will be one of your Associate Directors of Engineering this year. I like going to the gym, playing the tumbi, and coding.

**Comment by KallosP** [comment]
Hello, my name is Peter Kallos. I'm a 4th year CS major but transferred last year from Antelope Valley College. Some of my hobbies include playing guitar, soccer, and working on personal projects.

**Comment by riannalei** [comment]
Hello, my name is Rianna Lei (she/her), and I'm a 3rd-year CS major. I’m originally from Los Angeles, CA, and a fun fact about me is that I can make yummy einspanners! I also love going outside, exploring nature, and discovering new coffee shops in my free time.

**Comment by LeticiaLeon-Rodriguez** [comment]
Hi, my name is Leticia Leon (she/her), and I'm a 3rd year Computer Science major. I'm from Madera, CA, and a fun fact about me is that I have 13 rabbits (not on purpose, but extremely adorable). I love exploring SLO and trying new restaurants.

**Comment by Alec-0dell** [comment]
Hello everyone! My name is Alec Odell. I use He/Him/His pronouns. I am a second-year computer science major from Seattle Washington. Fun fact about me is that I am a citizen of the United Kingdom. Some of my hobbies include basketball, golf, skiing, backpacking, and more!

**Comment by ljbarker** [comment]
Hi all, my name is Logan Barker (he/him). I'm a 3rd year Computer Science major and blended program student following the AI/ML concentration from Roseville (near Sacramento, CA). A fun fact about me is that I've broken records for powerlifting (although they've been broken since unfortunately). Some hobbies of mine are playing soccer, gaming, and campus involvement (you should totally join CSAI as well).

**Comment by dionnegregorio** [comment]
Hi, my name is Dionne Gregorio (he/him). I am a 3rd year transfer studying Computer Science. I am from Los Osos, Ca, about 20 minutes from Cal Poly. I was born in the Philippines and immigrated here when I was 8-years-old. A fun fact about me is that I am a licensed barber in California. My hobbies include hiking, running (if I'm not injured), working out, camping, and playing video games (if I have free time) and working on my programming skills. I aspire to get involved on campus by joining clubs, graduating and becoming a software engineer or UI/UX designer.

**Comment by atkin0** [comment]
Hi I'm Atkin Rong (he/him). I'm from Elk Grove (close to Sacramento) and am a second year in Software Engineering. A fun fact is that I'm missing two teeth (idk why). One of my favorite hobbies is walking around and yapping with my friends. I also watched a bunch of shows over the summer.

**Comment by lamkin1** [comment]
Hello, my name is James Lamkin (he/him). I am a 4th year Computer Science student with a Data Science minor and I'm from Boise, Idaho. A fun fact is that I travel around the country for various competitive Spikeball tournaments. I'm also the Roundnet (Spikeball) club president this year, but while I'm not spiking I'm usually playing other sports or guitar.

**Comment by Ktigas** [comment]
Hello, my name is Kyrene Angelica Tigas (she/her). I am a 2nd year Computer Science Student and I am from Hercules, CA. A fun fact about me is that I lived in Singapore for eight years! My hobbies include hiking, running, traveling, and making/listening to music.

**Comment by Firazaba0369** [comment]
Hello, my name is Francisco Irazaba (He/Him). I am a 3rd-year Computer Engineering major from Apple Valley, CA. I have hobbies ranging from gyming, tennis, and skateboarding.

**Comment by Taliavivrett** [comment]
Hi! My name is Talia Vivrett (she/her). I am a second year Software Engineering major from Ashland, OR. A fun fact about me is that I am an identical twin. My main hobbies are lifting, tennis, hot yoga, and cooking.

**Comment by ChachaTeraSira** [comment]
Hey y'all,
My name is Khushkaranpreet Singh Grewal, from Central Valley, California. I'm a Computer Science transfer student starting fall 2024. Fun fact about me, I recently graduated from my Community College with 10 Associate Degree's and 5 Certifications. My hobbies include sketching, listening to music while going on solo drives, Playing CODM.

**Comment by timofey-lipunov** [comment]
Hey everyone, 
I'm Timofey Lipunov (Tim), coming from San Jose, California. I'm a current 3rd year Computer Science major, and a few hobbies that I have including snowboarding and cooking I also participate in judo and jiu jitsu competitions from time to time, and can be seen volunteering for red cross, either on campus or back in the bay area.

**Comment by samikshakarimbil** [comment]
Hi everyone! My name is Samiksha Karimbil and I use she/her pronouns. I'm an international student from Bangalore, India. I'm a 3rd year Computer Science major, and I really like to dance and watch TV shows. I also like to play board games and solve rubik's cubes

**Comment by kevinrutledge** [comment]
Hi everyone! My name is Kevin (he/him) and I am from San Jose, California. I'm a third year transfer Computer Science major, onboarding from De Anza-Foothill College. I am a bit of tourist when it comes to my hobbies as I recently am closing my chapter of Ballroom dancing/teaching. I am, however, transitioning more into social dances like Salsa which I currently find exhilarating. If there's spare time (which seems to be rarer and rarer these days) I like reading or watching shows/movies/anime/you-name-it. Otherwise, I enjoy the relaxing things in life -- walking my dogs, hikes, or going out to eat.

**Comment by ssasukek** [comment]
Hi everyone! My name is Kevin, i'm from Oakland, CA. Currently a third year Computer Science major and my hobbies are cooking and gaming. When i'm not doing any of those I usually watches kdrama and anime.

**Comment by arossi09** [comment]
Hello everybody! My name is Anthony and I'm from Salinas, CA. I am a 3rd year Computer Science major and I like to play the guitar and go to the gym. Some of my hobbies are sewing, animation, and reading novels/manga( I'm currently reading Vagabond, and it's really good).

**Comment by vmagdangal** [comment]
Hello! My name is Vinpatrik (he/him). I'm from Vallejo, CA, and I'm a fourth year Computer Science student. I love going out to photograph and observe wildlife, especially birds and whatever appears at the beach. I'm also really passionate about videogames in general and recently took an interest in building Gunpla models.

**Comment by koryyuen** [comment]
Hey guys! My name is Kory Yuen (he/him). I'm from San Gabriel, CA and I'm a second-year Computer Engineering student. A fun fact about me is that I'm left-handed. Some hobbies I have are playing video games, going to the gym, and watching sports (mainly basketball and football).

**Comment by johnnyjjue** [comment]
Hi everyone, I'm Johnny! I'm a 3rd year Mechanical Engineering major and I'm from Los Angeles, CA. Some things that I really enjoy doing includes going to the beach, the gym, and I also love cooking and baking lots of new foods.

**Comment by mattphanm** [comment]
Hi everyone! My name is Matthew Phan, and I am a 2nd year at Cal Poly SLO, majoring in Computer Science. I am from Los Angeles, CA. A fun fact about me is that I can make very good matcha lattes. Some of my hobbies are cooking and listening to music. Recently, I have been learning how to cook. Although I am a beginner, I am putting in the effort and am determined to improve to learn several recipes!

**Comment by Amogh76** [comment]
Hey everyone, my name is Amogh Arora. I'm from Fremont California and am a freshman Computer Science Major. A fun fact about me is that I can catch food in my mouth after throwing it up. Some of my hobbies are playing basketball and cooking. I've just recently started both but think I am improving quickly!

**Comment by emi-a-dinh** [comment]
Name: Emi Dinh She/Her
Hometown: Sacramento, 
Year & Major: I'm a 3rd year computer science student 
Fun Fact: I've been dyeing my hair since 1st grade that I don't remember the last time my hair was natural 
Interests: I like rock climbing

**Comment by Shonueld** [comment]
Hi everyone! My name is Sean Griffin and I'm from Simi Valley, California. I'm a third-year transfer student from Moorpark College studying Computer Science. Something interesting about me is that I have recently really gotten into studying geography and am trying to learn where every country is on the map. Now that I'm here in SLO, I definitely want to start taking up hiking as a hobby with all the trails around us!

**Comment by HenryHorse** [comment]
Hello! My name is Shayan Daijavad and I'm a 4th year CS major. I'm from Morgan Hill, California. A fun fact is that I lived in New York state for the first ten years of my life. My interests include exercising (yoga, lifting, running), reading, cooking, and playing video games.

**Comment by Rubensaghian** [comment]
Hello everyone! My name is Ruben Ramirez and I go by He/Him/His pronouns. I'm from LA and I am a 1st year CS major. A fun fact about me is that I like to make my own sourdough bread. My interests are Soccer (big Barcelona fan), skating, playing video games,  and I really love coffee.

**Comment by RyanSchmit** [comment]
Hi, my name is Ryan Schmitt (he/him/his). I am a 3rd year CS major. I'm from Long Island, New York and I like to surf, lift, and golf.

**Comment by AlexBloomer** [comment]
Hi, my name is Alex Bloomer and I go by He/Him/His pronouns. I am from Los Angeles, California and I'm a 1st year Computer Science Major. A fun fact about me is that I've been to Antarctica. I enjoy hiking, playing Spikeball, watching sports, and playing video games, and I'm trying to get into climbing.

**Comment by TylerHBE** [comment]
Hi, my name is Tyler Hickerson (He/Him), I'm from Santa Rosa, California and I'm a 1st year Computer Science Major. A fun fact about me is that I make an amazing pumpkin pie from scratch (garden pumpkins). Right now I'm really into music production and card games.

**Comment by clngo** [comment]
Hi! My name is Colin Ngo (he/him/his) from Los Angeles, California. I’m a third-year Computer Science major. Fun Fact: As someone who is left-handed and right-eye dominant, I like to think I use both sides of my brain pretty well! I enjoy running and swimming, and I'm looking to try out soccer soon.

**Comment by NolanKnievel** [comment]
Hey! My name is Nolan Knievel and I'm a 2nd year CS major from Seattle, WA. A fun fact about me is I'm related to Evel Knievel. In my free time, I love to work out, swim, camp, ski, and just be active in general.

**Comment by Matthew-Le-027** [comment]
Hi! My name is Matthew Le, and I am a 3rd year CS major from San Luis Obispo. One fun fact about me is that I played the piano for 5 years. In my free time, I enjoy doing physical activities such as playing basketball, pickleball, running, and hiking, but I also enjoy playing video games and watching anime.

**Comment by ian-ang-zhihan** [comment]
Hello! My name is ian and I'm a 3rd-year transfer majoring in Computer Science. I'm from the Bay Area. Fun fact about me is I grew up in Malaysia! During my free time, I enjoy playing Ultimate Frisbee and spending my time outdoors (or sometimes just alone in my room.. depends on what I'm feeling that day)

**Comment by kailaaquino** [comment]
Hi! My name is Kaila Aquino and I'm a 4th yr computer science major. I'm from Hercules, CA. A fun fact is that I enjoy spicy food! My hobbies are going to the gym, trying new restaurants w/ friends, and cooking.

**Comment by carboninl** [comment]
Hi! I'm Nick Carboni, from just outside Denver, CO. I am a second year Computer Science student, and am really into snowboarding--fun fact I broke my wrist doing it last winter. Hobby-wise I also enjoy hiking and exploring fun places in nature with my friends!

**Comment by tmacias21** [comment]
Hi! My name is Trace Macias and I'm from Claremont, California. I'm a freshman, majoring in Computer Science. A fun fact about me is that I ranked #7000 global in Clash Royale last year. My hobbies include ping pong, musical theater, running, and watching the Dodgers and the Seahawks.

**Comment by jjonathanlauu** [comment]
Hi everyone! My name is Jonathan Lau(he/him) and I am a third-year CS major from Arcadia, CA. A fun fact about me is that I am immortal in Valorant. Some of my interests are hooping, lifting, video games and hanging out with my friends. I am super excited to meet all of you and have a good year!

**Comment by Amend37** [comment]
Hi! my name is Anthony Mendoza I'm a third year CS major. I am from Richmond, CA, a fun fact about me is that I can play 3 different instruments (guitar, drums, saxophone). Some hobbies I have are learning new things on my own by doing DIY projects, playing music. another fun fact is I carried my immortal friend during a tournament in valorant.

**Comment by VH571** [comment]
Hi guys my name is Victor Herrera I'm a 3rd year computer major from San Bernardine CA and one of my hobbies include vinyl collecting and hiking.

**Comment by jaxonsprogis** [comment]
Hey there, my name is Jaxon Sprogis. I am currently a third year computer engineering student. I am from the Palm Springs area, specifically La Quinta. A fun fact about me is that I went to Greece this summer. I enjoy working out, hanging with friends, and thrifting.

**Comment by RandomCyberCoder** [comment]
Hi, my name is Emmanuel Delgado and I'm from Santa Rosa CA. I am a fourth year computer science major. A fun fact about me is that I am taking a volleyball and beach volleyball course this quarter. I enjoy running (especially trail running) and road biking around town. Eventually, I would like to bike to the nearby cities.

**Comment by mchn-4305** [comment]
Hey everyone my name is Micah Chen! I go by he/him and I'm from Salinas, CA. A fun fact about me is that I love to collect hobbies. These hobbies include surfing, rock climbing, and hiking!

**Comment by blwelsh** [comment]
Hi! My name is Brady Welsh (he/him) and I'm a second year CS Major from Philadelphia, PA. A fun fact about me is that I was in Yellowstone this summer. A few of my hobbies include tennis, running (picked it up this summer), and playing the piano.

**Comment by AlexsVargas-CS** [comment]
Hey everyone,
My name is Alex Vargas (He/Him) and I am currently a sophomore majoring in Computer Science (He/Him). I am also from Los Angeles, California! Some of my hobbies include swimming, playing basketball, reading, and playing video games.

**Comment by otvalenc** [comment]
Hi! My name is Otoniel Valencia (He/Him) and I am a second year Computer Engineering Major. I am from Ontario, California (not Canada...). A fun fact about me is that I lived in Mexico for 7 years. A couple of my hobbies are playing video games, listening to a wide variety of music, and watching films (currently wanting to watch interstellar because I've heard many great things).

**Comment by justinch4n** [comment]
Hi everyone! My name is Justin Chan (He/Him/His) and I am a third year Computer Science major. I am from Los Gatos, CA (which is west of San Jose!). Some things I like to in my free time are cooking and weightlifting! A fun fact about me is that I can do a cartwheel.

**Comment by tomaseuu** [comment]
Hello! My name is Thomas Le (He/Him/His) and I am a third year Computer Science major. I am from West Covina, California (basically in the 626).  A fun fact about me is that I can guess every BlackPink (kpop group) title song with just 15 seconds of listening to it. My hobbies are video-editing, filming, reading, doing what excites, and binging Korean dramas (currently watching: Welcome to Samdal-ri).

**Comment by KevinBeltran23** [comment]
Hey everyone,
My name is Kevin Beltran (He/Him) and I am a third year Computer Science major. I am from Santa Maria, CA. I love distance running, playing the piano, listening to music, hiking, and eating ice cream. My favorite ice cream flavor is churros and milk :)

**Comment by dmart297** [comment]
Hello! My name is Drew Marting (She/her) and I am a 2nd year computer science major. I am from San Diego and one fun fact about me is that I have two middle names (Torrey Lee). I love to read, play the drums, play video games, and bake! I used to play the drums when I was little and just picked it back up this summer!

**Comment by th0masF** [comment]
Hey everyone! My name is Thomas Flores (he/him/his) and I'm a 2nd-year Computer Engineering major from Patterson, California! A fun fact about me is I can do multiple character voices including Joe Swanson, Elmo, and Kermit the Frog. I love spending time outdoors whether it be hanging out at the river or hiking Bishops. I also enjoy working out and learning about the science behind it!

**Comment by colinbyang** [comment]
Hey everyone, my name is Colin Yang (he/him/his) and I'm a first year computer science major from Arcadia, California. 
A fun fact about me is that I enjoy playing soccer and running track. My main events for track are 100m and 200m and I'm hoping to walk onto the track team here. I enjoy working out and eating/trying new foods.

**Comment by MekaSid** [comment]
Hi! I am Sid Meka (he/him/his) and I am a third year computer science major from Pleasanton, California. A fun fact about me is that I play the piano and basketball. I also have recently picked up cooking after watching "The Bear."

**Comment by annahuang28** [comment]
Hi!! My name is Anna Huang (she/her) and I am from San Francisco. I am a 2nd year computer science major. A fun fact about me is that I am very spontaneous. My hobbies are gaming with friends, traveling, and going to concerts.

**Comment by tranguyen-grc** [comment]
Hi! I'm Tra Nguyen, a 2nd-year LAES major. I'm from Palo Alto, CA. In my free time, I like to read manga and do nails!

**Comment by amandayu255** [comment]
Hi, I'm Amanda Yu (she/her) and I'm from Oakland, CA. I'm a 3rd year CS major and my hobbies are video games and watching TV shows.

**Comment by quinniedapooh** [comment]
Hello everyone! I'm Quinton Wong (he/him), and I'm from Wailuku, HI. I'm a 4th-year Computer Science major with a Chinese minor. I'm a fraternal twin (younger by two minutes) and love running, knitting, and playing board games with friends.

**Comment by Vishnu-Alachi123** [comment]
Hi Everyone, My name is Vishnu Alachi(He/Him), I am from Kerala, India. I am a 2nd year Software Engineer major. A fun Fact about me is that I make custom Stickers. Some of my hobbies are playing basketball, soccer and drawing.

**Comment by hoikinlam** [comment]
Hello, my name is Hoikin Lam. I was born in Hong Kong, China and moved to the US when I was around 8. I'm a 3rd year computer science transfer from Los Angeles. A fun fact is that I play the bassoon. Some of my hobbies are playing video games, eating, going on adventures and seeking thrills.

**Comment by andyc88** [comment]
Hi everyone! My name is Andy Cai (He/Him), and I'm originally from San Francisco, California. I'm a 2nd year Software Engineering Major & a fun fact about me is that I was in China while COVID was starting. Some of my interests include listening to music, working out, and also working on and driving my car!

**Comment by Eclypsee** [comment]
Hi! My name is Evan Lee(he/his) from the Bay Area in California. I'm a second year computer engineering major. A fun fact about me is that I can write with both hands. I enjoy hiking, biking, and swimming.

**Comment by IsaacPressman** [comment]
Hi, my name is Isaac Pressman (he/him), I'm from the San Ramon, California which is part of the Bay Area. I'm a 1st year computer science major. A fun fact about me is that I can name all the flags of the world. My hobbies include playing basketball, going to the gym, and playing the piano.

**Comment by aleecodi** [comment]
Hi yall, my name is Aaron Lee. I use he/him pronouns. I have roots in both Taipei, Taiwan and Palo Alto, California. I'm a freshman in Software Engineering. A fun fact is I can play almost all saxophones except bass and soprano. My hobbies include climbing mountains(bullet climbed Mt.Fuji in the 1st week of this past September), playing video games(hard stuck bronze :) ), and listening to J-pop and Indie music. If you want to contact, insta: @aaronl623 or discord: crazymaniac0623 .

**Comment by aabro1325** [comment]
Hey, my name is Adarsh and I'm a 1st year Computer Science Major from San Jose, California. I love to play golf and watch football (both college and nfl).

**Comment by EthanSunjoonYang** [comment]
Hello! My name is Ethan (he/him) and I'm a first-year Computer Science major from Eugene, Oregon. A fun fact about me is that I'm bilingual (Korean and English). In my free time, I like to hike, play beach volleyball, listen to music, and take cool pictures. I'm also Ascendant 2 in Valorant.

**Comment by Galaxy25** [comment]
Name: Nathan St. John (He/Him)
Hometown: Portland, Oregon
Year & Major: I'm a first-year computer engineering student.
Fun Fact: I used to fence in high school.
Interests: I'm into weightlifting, hiking, cooking, reading classics, finding new music, game development, and much more.

**Comment by LambNNot** [comment]
Hi everyone! My name is Sam Phan and I am second Computer Science major. I am from San Diego, California and fun fact: I am on the Cal Poly Lion Dance Team! Some of my interests include badminton, writing, video games, and game design. Please hit me up if you play Legends of Runeterra or Eternal Return pls :D.

**Comment by Ctm31** [comment]
Hi! My name is Trotter McLemore (they/them), and I'm a second-year CS Major. I'm originally from Portland, OR, and my fun fact is that my dog at home is a St. Bernard who weighs more than me! Some of my hobbies include rock climbing, skiing, and reading.

**Comment by MatthewBlam** [comment]
Hey there, my name is Matthew Blam and I'm a first-year CS Major. I'm from Irvine, California and a fun fact is that I fence 🤺. I enjoy playing most sports, playing and making music, gym, hanging out, and coding.

**Comment by taikimorita** [comment]
Hey, my name is Taiki Jeffers and my pronouns are he/him. My hometown is Hoboken, NJ and I'm a first-year Computer Science major. One fun fact about me is that I like to unicycle, and I even unicycled the Five Boro Bike Tour in New York City back in May. Some of my other hobbies include rock climbing, skiing, volleyball, and playing Counter-Strike 2. My CS interests include software engineering, web development, Linux admin, and cybersecurity.

**Comment by 1kyler1** [comment]
Hello my name is Kyler (He/Him) and I grew up in Bellevue Washington. I am a second year computer science major and a fun fact about me is that I am bilingual and speak Spanish. In my free time I go to the gym, beach and play tennis.

**Comment by allysonstauffer** [comment]
Hi! My name is Ally (she/her) and I'm from Pleasant Hill, California! I'm a 1st year Computer Science major. One fun fact about me is that I've done ballet for 12 years! I love to sing (I'm in one of the Cal Poly Choirs) and listen to music!

**Comment by StearmanRubey** [comment]
Hello! My name is Stearman(He/Him) and I'm from Denver, Colorado. I'm a freshman in Computer Science. Something interesting about me is that I'm a silver palm Eagle Scout. My hobbies include playing video games, spending time with my dogs, drawing, and going for walks!

**Comment by darpank23** [comment]
Hi! My name is Darpan Khanna (he/him) and I'm from Roseville, California. I'm in my first year of computer engineering. Something interesting about me is that I lived in Singapore for 7 years! My hobbies include playing tennis, hanging out with friends, and playing Forza Horizon 5.

**Comment by Deep-Singh14** [comment]
Hello 👋
My name is Deep Singh (he/him) and I'm from Ripon, CA. I am a first-year CS major and something interesting about me is that I love driving and always plan to travel in a car. A hobby of mine is listening to music, so I love exploring genres and artists.

**Comment by jasonyu0705** [comment]
Name: Jason
Hometown: Guelph canada.
Year & Major: 3rd year computer engineer
Fun Fact: I am originally from Canada
Interests: I play badminton

**Comment by bbq27** [comment]
Hello! My name is Berfredd Quezon (he/him) and I'm from Long Beach, California. I'm a 2nd year Computer Engineering major and a cool fun fact about me is that I've had 12 different hair colors before. Some of my hobbies include playing video games, cooking, and going to concerts!

**Comment by leili2000** [comment]
Hello, my name is Leili Nelson. My pronouns are she/her and I'm from San Luis Obispo, California. I am a 2nd year computer science major. My favorite music group is Twice. I also love general semantics.

**Comment by Himeh12** [comment]
Hello all! My name is Jaime (Hi-Meh) Almeida and my pronouns are he/him. I am from Vallejo, California and I am a 4th year Computer Engineering major. I love playing basketball and lifting!

**Comment by Migue654** [comment]
Hey everyone, my name is Miguel Medina (he/him), I'm currently a first year software engineering major. Im from Tulare California, a fun fact about me is that I travel to Mexico every summer and going touring. I enjoy going on walks, working out, and learning new things.

**Comment by fabian2111** [comment]
Hi everyone, my name is Fabian Ballesteros-Limon (he/him) and I am currently a first year software engineering major. I am from Tulare, California and I am currently learning how to play the guitar. I enjoy hanging out with friends and doing cookouts with them.

**Comment by lalog12** [comment]
Hi everyone, my name is Eduardo Gallegos, and my pronouns are he/him. I'm from Oxnard, California. I'm a 4th-year Computer Engineering major, and my expected graduation date is December 2025. A fun fact about me is that my shoe size is 15 for men. I love to play basketball, hike, code, and play video games.

**Comment by chenyizhao14** [comment]
Hello! I'm Chenyi Zhao (she/her) from Fremont, California. I'm a 3rd year Computer Science major. I reached 1900 in chess when I was 11 and currently hold the WCM title (I haven't played professionally since). Some hobbies include badminton, baking, dancing, and playing video games.

**Comment by rachanaraju3** [comment]
Hi, I’m Rachana Raju (she/her) and I’m from San Jose. I’m a second year computer science major and I like to read and crochet in my free time!

**Comment by geethitarra** [comment]
Hi everyone!!! My name is Geethi Tarra (she/her). I am a freshman majoring in Computer Engineering from Cupertino, California. A fun fact about me is that I have four pet chickens. In my free time, I enjoy dancing, traveling, and petsitting.

**Comment by mcmike29** [comment]
Hello all! My name is Mike Campos (he/him) and I'm a 3rd year computer engineering major from Orange County, California. A fun fact about me is that I play bass, and some interests I have are music, boardgames and videogames, working out, reading, and coding for fun occasionally.

**Comment by Tylermhh** [comment]
Hello! My name is Min Hset Hlaing or Tyler (he/him) and I'm a 4th year computer science major from Yangon, Myanmar. I like to listen to music, play volleyball and play instruments when I'm free!

**Comment by shaaronl** [comment]
Hi! I’m Sharon Liang (she/her) and I’m a 3rd year computer science major. I’m from San Francisco and a fun fact about me is that I’ve never broken a bone. In my free time, I enjoy exploring new tech and experimenting with new cooking recipes. I also love editing videos, playing tennis, and making matcha lattes!

**Comment by SaurSum8** [comment]
Hello, my name is Saurish Suman (he/him) and I am a 1st year Computer Science major. I am an international student from India, but I have lived in the US previously. My main hobbies include programming, math, biking, and playing some video games.

**Comment by pragtoppo** [comment]
Hi! My name is Pragati Toppo (she/her) and I'm a 3rd year Software Engineering major. I'm an international student from Mumbai, India. A fun fact about me is that I've got to visit every Disneyland (with the exception of Disneyland Tokyo). My current hobbies include playing games on my old Nintendo DS which I never got to complete when I was younger (I highly recommend Ace Attorney) and dancing (I'm in the K-POP dance club, K2!).

**Comment by AgustinDimayuga** [comment]
Hi my name is Agustin Xocua and I'm from Santa Ana. I'm a first year computer science major. Something interesting about me is I work at Raising Canes. One of my favorite hobbies is playing Rainbow Six Siege.

**Comment by ncoronac** [comment]
Hi!! My name is Noemi Corona (she/her). I'm from Compton and I'm a second year Computer Science major. A fun fact about me is that I love going to Six Flags and my favorite ride is CraZanity. I've been getting into going to the gym recently and I'm currently taking a bowling class for fun!

**Comment by jonathanramirez4479** [comment]
Hi all, my name is Jonathan Ramirez (he/him) and I'm from King City, California.  I am a senior in computer science.  I enjoy trying new restaurants wherever I go. I have been loving the resident evil game series that I recently dove into.

**Comment by DashP06** [comment]
Hello, my name is Dash Pickrell (he/him). I'm a 1st year CS major from Seattle, WA. I like looking out for good food. In my free time, I enjoy working out and listening to music. Nice to meet you all.

**Comment by jasmiineng** [comment]
Hi! My name is Jasmine Ng (she/her/hers). I m a second-year Computer Engineering student from Alamo, CA. I enjoy going to the gym, hiking, and hanging out with my friends in my free time. A fun fact about me is that my comfort movie is Minions.

**Comment by bri8679** [comment]
Hi, my name is Briana Kirkman(she/her) and I am a 3rd year Computer Science Major from Murrieta, CA. I play violin in the school symphony and I like to dance.

**Comment by Edgarddragde** [comment]
Name: [Your Name] (Pronouns)
Hometown: [City, Country]
Year & Major: [e.g "I'm a 3rd year transfer in Computer Science" or "I'm a freshman in Graphic Communications"]
Fun Fact: [Something interesting about you, e.g., "I play the ukulele!" or "I can solve a Rubik's cube in under a minute!"]
Interests: [Your hobbies, goals, and aspirations eg., "I love hiking and the outdoors." or "I'm Radiant in Valorant!" ]\

Hi, my name is Edgard Aviles (He/Him/His) and I am a 2nd year Computer Science major from Santa Ana. I love playing Rainbow Six Seige and solving the Rubik's cube; also I love running.

**Comment by suesuee** [comment]
Hi, My name is Sue. I am a third-year transfer, majoring in computer science. I used to play violin back in my high school's orchestra. My hobbies include reading books, swimming, listening to music, watching sitcoms, and talking to my friends/family members.

**Comment by JavierMB03** [comment]
Hello. My name is Javier (He/Him/Hims), and I am from Oxnard, about an hour away from LA. I'm a third-year transfer in Computer Science, and I currently do not have a concentration. A fun fact about me is that I can whistle four different ways. I enjoy playing chess and going to the gym during my free time.

**Comment by Anivishy** [comment]
Hey! My name is Ani (he/him), and I am from Sammamish, Washington. I am a first year Computer Science major. I love to go to the gym and also shoot competitive Olympic recurve archery.

**Comment by anthonythang1** [comment]
Hi everyone,

My name is Anthony Thang (he/him), I'm a CS transfer student from Sacramento, CA! One fun fact about me is that I've never seen any High School Musical Movies! I like to stay active and lift weights or play basketball but also love working on cars to make them go fast!

**Comment by ltrush** [comment]
Hi, my name is Luke (he/him), and I'm from just north of San Francisco (Marin County). I'm a third year computer engineering major. A fun fact about me is that I went to Beyond Wonderland in March! I had a wonderful time. I love to read (currently reading Three Body Problem series) and play video games (R6 and Valorant even though I hate R6). I also would love to one day become a film director.

**Comment by camilayeremin** [comment]
Hi, my name is Camila (she/her/hers), and I'm from San Jose California. I am a third-year LAES major, with concentrations in computer graphics and art.  One fun fact about me is that I've played [almost] all of the TLOZ games! My biggest hobbies are probably reading and drawing!

**Comment by hchencp** [comment]
Hi, I'm Henry (he/him), and I'm from Oakland, California. I'm a third-year computer science major. One of my favorite hobbies is experimenting with different tea leaves to create unique drinks. A fun fact about me is that I enjoy collecting trading cards.

**Comment by TheGreatKiwi** [comment]
Hi, I'm  Keira (she/her). I'm from the bay area and I'm a first year in computer science. A fun fact about me is that I have two bunnies, named Loki and Clover. I like to do jiu jitsu and I like to bake.

**Comment by jacobponce** [comment]
Hello, my name is Jacob and I am from San Ramon California. I am a 3rd year computer engineering student. Something interesting about me is I have played golf since I was 5. In my free time, I like to play golf and video games.

**Comment by derrickph33** [comment]
Hello, my name is Derrick Phan and I am a 3rd year Computer Science major from Berkeley, California. A fun fact about me is that I compete in Table Tennis at the national level. Some of my hobbies are playing golf and basketball, working out, and playing video games.

**Comment by westonsam** [comment]
Hey everyone! My name is Sam Weston and I am a 2nd year Electrical Engineering student from Roseville, California. A fun fact about me is that I am on the Cal Poly rugby team. I love tinkering, making music, and working out.

**Comment by DoanCal** [comment]
Hello everyone, My name is Tu Doan (He/Him) and I am from Oakland, California. I am a 2nd year Computer Science major and a fun fact is that I'm champion in r6. I love going to the gym, playing games, and spending time out with friends!

**Comment by akshatk1001** [comment]
Hi everyone! I'm Akshat Khandelwal (he/him) and I'm from Dublin, California - about 30 minutes from SF. I'm a first year Computer Science major and am particularly interested in Computer Vision. A fun fact about me is that I am a sneakerhead and have a pretty big collection of shoes back home. I love playing any and all sports, hiking, reading, and of course coding!

**Comment by markmcguire011** [comment]
Hi everyone, my name is Mark McGuire (he/him). I'm a 2nd year CS major and Math minor from Menlo Park, CA. My fun fact is that I was an all around state champion gymnast for California when I was 9. I love basketball and staying active, playing the piano, reading old books, and traveling.

**Comment by karthik0605** [comment]
Hello everyone, my name is Karthik Balaji (he/him). I am a 2nd year CS major from San Jose, CA. My fun fact is that I know how to juggle. I love playing badminton and chess, and also love to explore nature and travel.

**Comment by wliu4040** [comment]
Hello, my name is William Liu (he/him). I'm a 2nd year CS student from San Francisco. A fun fact about me is that I've been learning the piano. I like to play video games, go fishing, and play the piano.

**Comment by Velevynn** [comment]
Hello! My name is Alexander Specht (he/they). I am a fourth year Computer Science major from Sacramento, CA. A fun fact is that I have a porcelain tooth, and I like to dance, read, and listen to music!

**Comment by mio-naka** [comment]
Hello! My name is Mio Nakagawa (she/her) and I am a 4th year computer science major from Kansas! A fun fact is that I was raised in Kyoto, Japan, and my hobbies include going to workout classes, the beach, reading, and watching tv shows/movies.

**Comment by anissasoungpanya** [comment]
Hi! My name is Anissa Soungpanya (she/her) and I'm a 3rd year computer science major from the bay area! A fun fact is that I have been to Greece and Italy! My hobbies recently have been cooking, painting and thrifting.

**Comment by tdefl** [comment]
Hello! My name is Alfred Tran (he/him). I'm a 4th year transfer from Sacramento, CA., studying Computer Science. A fun fact is that I participated in a self-driving car competition in Canada over the summer.

**Comment by ewong128** [comment]
Hi everyone! My name is Emily Wong(she/her) from Los Angeles, CA. I'm a 4th year majoring in Computer Science. Fun fact about myself is that I am a really big foodie and I like to cook. Also, I like to play tennis and badminton.

**Comment by qarteu** [comment]
Hi. My name is Kartik Malunjkar (he/him), and I'm from Cupertino, CA. I am a first-year majoring in computer science. A fun fact about me is that I was an Apex Legends professional.

**Comment by ChristopherYoeurng** [comment]
Hi, my name is Chris. I'm from LA and I'm a third-year CS major. Fun fact about me is that I like to sing :)

**Comment by lia-glitch** [comment]
Hi my name is Lia, Im from the Bay Area and a first year CS major:)

**Comment by angelwu1** [comment]
Hi everyone! My name is Angel and I'm from Brooklyn, New York. I'm currently a sophomore studying Computer Science. My fun fact is that I can solve a Rubik's cube in under a minute! My hobbies include traveling, going out to try new restaurants and chronically checking my Beli :)
